docs/help/git_webhooks.md

```diff
@@ -0,0 +1,30 @@
+## Webhooks
+
+1. Generate a SSL certificate for the REST API. TODO: figure out the right
+   way to do that.
```

Comment:
Any example anywhere on how to do that? I begun with Limnoria's instructions (that I wrote) ending up with bitbot.pem that seems wrong.

Reply:
`$ openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365` e.g.

Reply:
Thanks, I will add it :)